CHANGELOG

```diff
@@ -0,0 +1,114 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on
```

Comment:
I thought you will just add it to the commit message.
It is a bit strange for a user to read this first (if he is reading this file, most likely he is interested in the changes and not its format).

Just consider this, it is not a must.

Reply:
Or we can mention all this in the README next to the link instead of here.